tool/gocross/toolchain.go

```diff
@@ -112,9 +112,18 @@ func ensureToolchain(cacheDir, toolchainDir string) error {
 		return err
 	}
 
-	if err := downloadCachedgo(toolchainDir, wantRev); err != nil {
-		return err
+	isLocalToolchain := len(wantRev) > 0 && wantRev[0] == '/'
```

Comment:
filepath.IsAbs?

Reply:
Or if you want consistency then add comment.

Reply:
Oh yeah! Done, thanks.